src/main/resources/Beans.xml

```diff
@@ -0,0 +1,17 @@
+<?xml version = "1.0" encoding = "UTF-8"?>
+
+<beans xmlns = "http://www.springframework.org/schema/beans"
+       xmlns:xsi = "http://www.w3.org/2001/XMLSchema-instance"
+       xsi:schemaLocation = "http://www.springframework.org/schema/beans
+   http://www.springframework.org/schema/beans/spring-beans-3.0.xsd">
+
+    <bean id = "airplane1" class = "com.endava.exercise.Airplane">
```

Comment:
having "1" in id value is anti pattern.
What do you think, why?

Reply:
I must admit that it does look wrong :D and I google it, but I couldn't find "officially" why. I suppose that it is ambiguous and non-descriptive - later when using the beans, what is the difference between "airplane1" and some other "airplane2"?

I have renamed it to be more descriptive.

Reply:
"...1" in the name only make sense if there are "....2" , "....3" and so on, but that's not the right way to get multiple airplane instances.
Ability of a component to be unique / singular or to be non-unique / plural is handled with bean scopes (Singleton and Prototype) , it shouldn't be handled by bean names / id's ("airplane1", "airplane2" etc ). 
You could have left this bean to be id="airplane" but made it a Prototype Scope which ensures that you can create many different instances of airplanes.

What you did to fix this is acceptable ( but is somewhat a work-around ) , you used specific plane type, in which case it's fine to have a Singleton.